cmake/Hunter/init.cmake

```diff
@@ -1,12 +1,10 @@
 
-if(EVMC_TESTING OR EVMC_TEST_TOOLS)
-    set(HUNTER_CONFIGURATION_TYPES Release
-        CACHE STRING "Build type of the Hunter packages")
+set(HUNTER_CONFIGURATION_TYPES Release
+    CACHE STRING "Build type of the Hunter packages")
 
-    include(HunterGate)
+include(HunterGate)
```

Comment:
How is this link resolved?

Also can you move the whitespace changes to the first commit which introduces the file?

Reply:
It's in `cmake/cable` which is added to CMake's module path.

What whitespace changes?

Reply:
The second commit removes spaces from the file added in the first commit.

Reply:
It removes `if()` so the indentation changes.